purchase_order_secondary_unit/tests/test_purchase_order_secondary_unit.py

```diff
@@ -61,14 +61,18 @@ def test_purchase_order_01(self):
             self.assertEqual(line.product_qty, 7.0)
             # Test onchange product uom
             line.secondary_uom_qty = 3500.0
+            self.assertEqual(line.product_qty, 2450.0)
             line.product_uom = self.product_uom_gram
-            self.assertEqual(line.secondary_uom_qty, 3.5)
+            self.assertEqual(line.product_qty, 2450000.0)
+            self.assertEqual(line.secondary_uom_qty, 3500.0)
```

Comment:
I am concerned about this change: the test was expecting `line.secondary_uom_qty == 3.5`, now it happens that `line.secondary_uom_qty == 3500`.
Usually when this happens it means that there might be something to be fixed in the code, or some cache value to be reset in the test, but the test's assertions shouldn't be change in a migration.

Or maybe are you saying that the test was asserting something wrong already in `15.0`?

Reply:
I'm not sure why these changes were made, maybe @yajo could help us, as original change (at least partially) was made by him, if I'm not wrong